lean4-info.el

```diff
@@ -137,18 +144,20 @@
     (lsp-request-async
      "$/lean/plainGoal"
      (lsp--text-document-position-params)
-     (-lambda ((_ &as &lean:PlainGoal? :goals))
+     (-lambda ((x &as &lean:PlainGoal? :goals))
        (setq lean4-goals goals)
-       (lean4-info-buffer-redisplay))
+       (lean4-info-buffer-redisplay-debounced lsp-buffer-uri))
```

Comment:
I don't know if threading this state throughout the computation is idiomatic. Perhaps there is a nicer way to retrieve the URI that corresponds to a Goal/PlainGoal?

Reply:
Is this case different from the `lean4-goals/term-goal` `defvar`s?